tasks/16_repeat-string/repeat-string.js

```diff
@@ -0,0 +1,9 @@
+export function repeatStringNumTimes(str, num) {
+  if (num < 1) return '';
```

Comment:
Why not `(num < 0 )`?

Reply:
Actually it turns out that the `if` statement isn't even needed so I optimized the code so it works without it.